test/python/apps/ethereum.py

```diff
@@ -1,16 +1,38 @@
-from ragger.utils import create_currency_config
+from typing import Optional
+import struct
 from ragger.bip import pack_derivation_path
 
+
+def get_sub_config(ticker: str, decimals: int, chain_id: Optional[int] = None) -> bytes:
+    cfg = bytearray()
+    cfg.append(len(ticker))
+    cfg += ticker.encode()
+    cfg.append(decimals)
+    if chain_id is not None:
+        cfg += struct.pack(">Q", chain_id)
+    return cfg
+
+
+def create_currency_config(main_ticker: str,
+                           application_name: str,
+                           sub_config: bytes = bytes()) -> bytes:
+    cfg = bytearray()
+    for elem in [main_ticker.encode(), application_name.encode(), sub_config]:
+        cfg.append(len(elem))
+        cfg += elem
+    return cfg
+
+
 ETH_PATH = "m/44'/60'/0'/0/0"
+ETC_PATH = "m/44'/61'/0'/0/0"
```

Comment:
What is the impact of this change on ETC test ?

Reply:
They seem to still pass, and it's better to have the tests use the real Ethereum Classic derivation path rather than the Ethereum one anyway :+1: